### DISCUSSION TITLE
Adding colors to mainline BN mods?

### DISCUSSION THREAD

**Discussion by thelonestander** [cataclysmbn/Cataclysm-BN#2099] [General]
Basically, I think it would be a good idea to color code the mainline BN mods. I think it would be nice for organization. 

![image](https://user-images.githubusercontent.com/82045140/197202676-74b19bd6-f79a-463c-a9bb-d89ad9a1fd10.png)

I think C.R.I.T. should be red, aftershock should be yellow. 

Balancing mods I think should be green. ![Screenshot (1218)](https://user-images.githubusercontent.com/82045140/197202996-bc6b70c1-54bf-45d3-be9c-960ca1b659e9.png)

Black listed mods could be dark green. 

Just an idea for some mod organization, and since only a few 3rd part mods use colors this could be nice for showing at a glance what mods ae base BN.

**Comment by olanti-p** [comment]
Such change will require forbidding modders from specifying custom colors of their mod's name, otherwise any semblance of organization will be lost behind a multitude of attention-grabbing highlights.

> Balancing mods I think should be green. 
> Black listed mods could be dark green.

Ideally we'd have a single list split by categories instead of the obnoxious subcategories within nested `Default`/`Blacklist`/`Balance` tabs we have right now. Then such division won't be needed, as each type will be under proper category header.
 
Here's an alternative idea - use 4 colors in total:
1. Mainline mods
2. External mods
3. Mainline mods overridden by external mods
4. Obsolete mods
5. (Potentially) "Bridge" mods (idea from https://github.com/CleverRaven/Cataclysm-DDA/issues/60351 )

This should be enough to figure out which mods come from BN and which - from other sources while not overloading the player with information.

**Reply by Zlorthishen** [reply]
>(Potentially) "Bridge" mods (idea from [Idea] Automatic glue mod management CleverRaven/Cataclysm-DDA#60351 )

kevingranade closed this as [completed] on **Oct 23, 2022**

**Comment by Zlorthishen** [comment]
>C.R.I.T. should be red

I implemented this in my current CRIT Renewal pr https://github.com/cataclysmbnteam/Cataclysm-BN/pull/2287

[Really Dark Skies](https://github.com/Zlorthishen/Really_Dark_Skies) will be using a variety of colors, depending on which 'major revision' of the mod is being used. Last color was light green, current color is light blue.
